### DISCUSSION TITLE
How to change reclaimPolicy without reinstalling everything?

### DISCUSSION THREAD

**Discussion by MirtoBusico** [longhorn/longhorn#10102] [Q&A] [answered]
Hi all,
I installed Longhorn using the helm chart with this command
```
 helm install longhorn longhorn/longhorn --namespace longhorn-system --create-namespace --version 1.6.1
```

This installs Longhorn with "**reclaimPolicy: Delete**"

Now I need to change it with "**reclaimPolicy: Retain**"

How can I do this without reinstalling Longhorn?

**Comment by derekbit** [comment]
Do you mean the reclaimPolicy of the default storageclass `longhorn`?

**Reply by MirtoBusico** [reply]
Yes

**Reply by MrMYHuang** [reply, marked as answer]
This works for me:
```kubectl -n longhorn-system edit configmaps longhorn-storageclass```
then find "reclaimPolicy" and set it to "Retain."
